README.md

```diff
@@ -16,27 +16,27 @@ Specify the following dependency in your [Carthage](https://github.com/Carthage/
 
 ```cartfile
 # Latest stable release
-github "mapbox/mapbox-directions-swift" ~> 2.6
+github "mapbox/mapbox-directions-swift" ~> 2.7
 # Latest prerelease
-github "mapbox/mapbox-directions-swift" "v2.8.0-rc.1"
+github "mapbox/mapbox-directions-swift" "v2.8.0-rc.2"
 ```
 
 Or in your [CocoaPods](http://cocoapods.org/) Podfile:
 
 ```podspec
 # Latest stable release
-pod 'MapboxDirections', '~> 2.6'
+pod 'MapboxDirections', '~> 2.7'
```

Comment:
update the latest stable version to `2.7.0` in `README.md`

Reply:
thanks for catching this shan!